sdk/core/azure-core/inc/azure/core/http/policy.hpp

```diff
@@ -387,7 +405,7 @@ namespace Azure { namespace Core { namespace Http {
     /**
      * @brief #Azure::Core::Http::Internal::ValuePolicy options.
      */
-    struct ValuePolicyOptions
+    struct ValueOptions
```

Comment:
`ValueOptions`? doesn't sound like a descriptive name

Reply:
I have created an issue to remove this policy from Core #1767 .

The reason that policy was created is to help the protocol implementation details from an sdk service.  An sdk client should maintain a policy like such but Coro should now encourage that